packages/hemera-mongo-store/test/index.spec.js

```diff
@@ -79,6 +68,38 @@ describe('Hemera-mongo-store', function () {
     })
   })
 
+  it('create with extended json', function (done) {
```

Comment:
Would be great to split the test suites in seperate files `ejson.spec.js`, `index.spec.js`